app/services/articles/feeds/basic.rb

```diff
@@ -19,6 +19,9 @@ def default_home_feed(**_kwargs)
         return articles unless @user
 
         articles = articles.where.not(user_id: UserBlock.cached_blocked_ids_for_blocker(@user.id))
+        if (anti_tags = @user.cached_antifollowed_tag_names).any?
```

Comment:
Maintaining variable names like `anti_tags` and `hidden_tags` for what I understand to be the same concept could potentially cause confusion - what do you think about using `hidden_tags` for newer variables and we can come back and later rename some variables like `cached_antifollowed_tag_names`.